cupyx/scipy/fftpack/fft.py

```diff
@@ -1,3 +1,4 @@
+from numpy import prod
```

Comment:
please add a blank line between the 3rd party (numpy) import and the cupy imports. (I believe this is the convention used by CuPy)

Reply:
You're right, thanks for reminding me CuPy's convention.